signup.html

```diff
@@ -21,33 +22,35 @@
     <!--This is used for the drop down menu-->
     <div id="header-placeholder"></div>
 
-<!--Main section for signing up
+    <!--Main section for signing up
     Allows to choose between client or administrator to log in-->
     <main>
         <section class="signup-box">
             <h1>Sign Up</h1>
-            <p>Would you like to sign up as a client or an admin?</p>
-            <a href="signup-client.html"><button>Client</button></a>
-            <a href="signup-admin.html"><button>Admin</button></a>
-        </section>
-    </main>
-
-
+            <p>Create a new account</p>
+            <div id="error" style="color: red;"></div>
+            <form id="client-sign-up-form">
```

Comment:
we have to choose between theses 2, Mariana had done 2 files for signing up one admin one client @Mariana-bs @victorrf2002

Reply:
is it only for client??